packages/app/src/debug/DebugResults.vue

```diff
@@ -9,31 +9,34 @@
       :total-skipped="results.totalSkipped"
       :total-pending="results.totalPending"
       :order="['PASSED', 'FAILED', 'SKIPPED', 'PENDING']"
+    />
+    <div
+      v-if="(results.totalFlakyTests && results.totalFlakyTests > 0)"
```

Comment:
Minor: I think we can just say

```suggestion
      v-if="results?.totalFlakyTests"
```

here. If `results.totalFlakyTests` is `null`, `undefined`, or `0` then this will evaluate to `false`. If it's greater than 0 then it will be `true`. No worries if you want to be more explicit

Reply:
Good catch! I believe the extra conditional is unnecessary and verbose as you've pointed out. Will make the change